apps/judicial-system/api/src/app/modules/backend/backend.service.ts

```diff
@@ -16,6 +16,8 @@ import {
 } from '../case'
 import { CaseListEntry } from '../case-list'
 import { Defendant, DeleteDefendantResponse } from '../defendant'
+import { CivilClaimant } from '../defendant/models/civilClaimant.model'
+import { DeleteCivilClaimantResponse } from '../defendant/models/deleteCivilClaimant.response'
```

Comment:
Please follow the established include pattern by using the defendant barrel file.